### PR TITLE
(PC-5961) Ajout demande géolocalisation sur la 3ème page des tutoriels

### PR DESCRIPTION
**Pull request by BenStirrup**
Link to JIRA ticket: https://passculture.atlassian.net/browse/PC-5961

## Checklist

I have:

- [x] Made sure the title of my PR follows the convention `[PC-XXX] <summary>`.
- [x] Made sure my feature is working on the relevant real / virtual devices.
- [x] Written **unit tests** for my feature.
- [ ] Added new reusable components to the AppComponents page and communicate to the team on slack
- [ ] Added a **screenshot** for UI tickets.
- [ ] Attached a **ticket number** for any added TODO/FIXME \
       (for tech tasks, give the best context about the TODO resolution: what/who/when).

## Screenshots

| \*iOS - [Phone name] | \*Android - [Phone name] | Tablet - [Phone name] | Little - [Phone name] |
| -------------------- | :----------------------: | --------------------: | --------------------: |
|                      |                          |                       |                       |

### CODE REVIEW

File: __mocks__/lottie-react-native.tsx [hunk 1]
```diff
@@ -1,9 +1,15 @@
...
-} 
+import ActualLottieView from 'lottie-react-native'
+
+type Props = {
+    testID: string
+}
+
+const LottieView = React.forwardRef<ActualLottieView, Props>(function LottieView(props, _forwardedRef) {
+    const displayNameForTest = `${props.testID}-Lottie-Mock`
+    return <View testID={props.testID}>{displayNameForTest}</View>
```
Comment (blixit) [inline __mocks__/lottie-react-native.tsx:12]: il ne râle pas que tu mettes une chaine de caractères dans une View et non dans un Text ?

**Review by blixit — APPROVED** [review]

**Review by DanielBaudry — APPROVED** [review]

File: __mocks__/lottie-react-native.tsx (hunk 1 above)
Comment (BenStirrup) [reply __mocks__/lottie-react-native.tsx:12]: Bonne remarque. Je peux te dire qu'il n'y a pas de warning en tout cas :)

File: src/libs/geolocation/requestGeolocPermission.android.ts [hunk 2]
```diff
@@ -1,8 +1,8 @@
 import { PermissionsAndroid } from 'react-native'
 
-export const requestGeolocPermission = async (onGranted: (granted: boolean) => void) => {
-  const granted = await requestGeolocPermissionSystem()
-  if (granted) onGranted(true)
+export const requestGeolocPermission = async () => {
+  const isGranted = await requestGeolocPermissionSystem()
+  return isGranted
 }
```
Comment (blixit) [inline src/libs/geolocation/requestGeolocPermission.android.ts:6]: on pourrait l'écrire plus simplement:
```ts
export const requestGeolocPermission = requestGeolocPermissionSystem
```

File: src/libs/geolocation/requestGeolocPermission.ios.ts [hunk 3]
```diff
@@ -1,8 +1,8 @@
 import Geolocation from 'react-native-geolocation-service'
 
-export const requestGeolocPermission = async (onGranted: (granted: boolean) => void) => {
-  const granted = await requestGeolocPermissionSystem()
-  if (granted) onGranted(true)
+export const requestGeolocPermission = async () => {
```
Comment (blixit) [inline src/libs/geolocation/requestGeolocPermission.ios.ts:3]: export const requestGeolocPermission = requestGeolocPermissionSystem

File: src/libs/geolocation/requestGeolocPermission.android.ts (hunk 2 above)
Comment (BenStirrup) [reply src/libs/geolocation/requestGeolocPermission.android.ts]: yes

File: src/libs/geolocation/requestGeolocPermission.ios.ts (hunk 3 above)
Comment (BenStirrup) [reply src/libs/geolocation/requestGeolocPermission.ios.ts]: yes

**Pull request merged**